modules/@apostrophecms/piece-type/index.js

```diff
@@ -745,6 +739,7 @@ module.exports = {
           {
             type: 'major',
             label: 'New ' + self.options.label,
+            // TODO: fully deprecate `insertViaUpload`
```

Comment:
Our image uploader is custom built so doesn't need this anymore. Our file uploader will likely get its own treatment as well, I'd imagine.